packages/react-reconciler/src/__tests__/ReactIncrementalErrorHandling-test.internal.js

```diff
@@ -256,56 +256,6 @@ describe('ReactIncrementalErrorHandling', () => {
     expect(ReactNoop.getChildren()).toEqual([span('Everything is fine.')]);
   });
 
-  it('on error, retries at a lower priority using the expiration of higher priority', () => {
```

Comment:
Intentionally removed this behavior; I'll add another test to describe what the new semantics are.